### PR TITLE
Added Q3 2024 Web3j Report

### PR DESCRIPTION
**Pull request by NickSneo**
Added Q3 2024 report for Hyperledger Web3j
Also I think last report was supposed to be Q2 which wrongly got posted under Q3, so changed the name of that file to Q2

### CODE REVIEW

File: project-reports/2024/2024-Q2-Hyperledger-Web3j.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: 2024 Q2 Hyperledger Web3j
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-Web3j.md:3]: Why are we changing the Q2 report?

File: project-reports/2024/2024-Q3-Hyperledger-Web3j.md
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Web3j.md:1]: Similarly, why are we changing the Q3 report?

I would expect that we want to submit the Q4 report. If this is what we are trying to do, please note that we have moved to the https://github.com/lf-decentralized-trust/governance repo for Q4 reports.

File: project-reports/2024/2024-Q2-Hyperledger-Web3j.md (hunk 1 above)
Comment (NickSneo) [reply project-reports/2024/2024-Q2-Hyperledger-Web3j.md:3]: I have renamed the previous Q3 to Q2, I thought it was wrongly named

File: project-reports/2024/2024-Q3-Hyperledger-Web3j.md
Comment (NickSneo) [reply project-reports/2024/2024-Q3-Hyperledger-Web3j.md:1]: Oh my bad I didn't knew this, in this event - https://lists.lfdecentralizedtrust.org/g/web3j/viewevent?eventid=2456180
It is linked to current repo.

**Comment by NickSneo** [comment]
Added the report in correct repo - https://github.com/LF-Decentralized-Trust/governance/pull/49

**Pull request closed without merging**